web/src/portico/landing-page.ts

```diff
@@ -129,9 +187,11 @@ $(() => {
 
     if (window.location.pathname === "/team/") {
         assert(page_params.page_type === "team");
-        const contributors = page_params.contributors;
-        delete page_params.contributors;
-        render_tabs(contributors);
+        if (page_params.contributors) {
+            const contributors = page_params.contributors;
+            delete page_params.contributors;
+            render_tabs(contributors);
+        }
```

Comment:
Why is this an `if` rather than an assert?

Reply:
Changed it to `assert` as I wasn't able to think of any case where `page_params.contributors` should be undefined in this case.